lib/schemas/config.js

```diff
@@ -0,0 +1,31 @@
+const yup = require('yup');
+
+const configSchema = yup.object().shape({
```

Comment:
still working on the tests for this...

Reply:
done: https://github.com/rewardops/rewardops-sdk-node/pull/46/files#diff-dab480eedb2fe55653dda98093b822ab